lms/static/scripts/frontend_apps/errors.ts

```diff
@@ -16,6 +16,7 @@ export type LTILaunchServerErrorCode =
   | 'd2l_student_not_in_group'
   | 'canvas_api_permission_error'
   | 'canvas_file_not_found_in_course'
+  | 'canvas_page_not_found_in_course'
```

Comment:
I think these are usually kept alphabetically ordered (at least among the same LMS, because D2L ones are above Canvas ones 😅 )